src/ast_408.ml

```diff
@@ -145,6 +145,8 @@ module Parsetree = struct
       ptyp_attributes: attributes; (* ... [@id1] [@id2] *)
     }
 
+  and typ = core_type
```

Comment:
Why does this PR modify this file?

Reply:
This is so that `gen_copy` generate the following functions automatically.
```
let copy_cases x = List.map copy_case x
let copy_pat = copy_pattern
let copy_expr = copy_expression
let copy_typ = copy_core_type
```

I think theses functions were previously manually added.

Reply:
ack